packages/app/components/nft-dropdown.tsx

```diff
@@ -96,39 +97,18 @@ function NFTDropdown({
   );
   //#endregion
 
-  const openModal = (modal: string) => {
-    const as = `/nft/${nft?.chain_name}/${nft?.contract_address}/${nft?.token_id}/${modal}`;
-
-    router.push(
-      Platform.select({
-        native: as,
-        web: {
-          pathname: router.pathname,
-          query: {
-            ...router.query,
-            chainName: nft?.chain_name,
-            contractAddress: nft?.contract_address,
-            tokenId: nft?.token_id,
-            [`${modal}Modal`]: true,
-          },
-        } as any,
-      }),
-      Platform.select({
-        native: as,
-        web: router.asPath.startsWith("/nft/") ? as : router.asPath,
-      }),
-      Platform.select({
-        native: {},
-        web: { scroll: false },
-      })
-    );
-  };
-
   const isShareAPIAvailable = Platform.select({
     default: true,
     web: typeof window !== "undefined" && !!navigator.share,
   });
 
+  const viewOnOpenSea = () => {
+    // Todo: maybe need to use token_id from backend.
+    const token_id = "1";
+    const link = `https://opensea.io/assets/matic/${nft?.contract_address}/${token_id}`;
+    Linking.openURL(link);
```

Comment:
does it work without `token_id` from backend?

Reply:
yeah, the backend's `token_id` always is 0, but OpenSea requires a valid(it will be greater than 0).
maybe no one claim when a user has dropped, so view the first claim NFT on OpenSea I think it works at least.